test/quantization/test_quantize_fx.py

```diff
@@ -1458,10 +1499,17 @@ def forward(self, x):
 
         # This model is not executable since we just put all ops
         # in the same forward
-        m = M().eval()
+        m = M().train()
         # nothing to fuse so skipping the fuse step
-        qconfig_dict = {'': default_qconfig}
-        prepared = prepare_fx(m, qconfig_dict)
+        qconfig_dict = {'': default_qat_qconfig}
+        prepared = prepare_qat_fx(m, qconfig_dict)
```

Comment:
What happens for the case of PTQ? We should not be inserting observers in that case. Is that being tested?

Reply:
we'll insert fixedqparamfakequant in that case as well

Reply:
I can add a test for ptq